src/_events/talking-to-alexa/description.md

```diff
@@ -0,0 +1,7 @@
+# Abstract
+In this talk we look into how we build a conversation with the Amazon Echo and Alexa. We'll look at the design, configuration and (a little) code that goes into building that natural back and forth, how we can use the abilities of the device to enhance the results, and how different APIs can be used to help enhance the experience.
```

Comment:
Try not to use the word "enhance" twice

Reply:
Not my text. So we'll not be changing that.

Reply:
:+1: